.github/workflows/check.yaml

```diff
@@ -20,22 +20,24 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - name: Cache go packages
-        uses: actions/cache@v4
+      - name: Run tests
+        run: go test -race ./...
+
+
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
         with:
-          path: |
-            ~/go/pkg/mod
-            ~/.cache/go-build
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}-
-            ${{ runner.os }}-go-
+          go-version: '1.23.x'
 
       - name: Run tests
-        run: go test -race ./...
+        run: go test ./...
```

Comment:
If you are running the go test with coverage in next step, so you don't need this "go test" step here

Especially as you move go test to "gotest"

Reply:
You're right. I haven't noticed it. Thanks